src/SprykerEco/Zed/Optivo/Business/OptivoBusinessFactory.php

```diff
@@ -1,86 +1,211 @@
+
+    /**
+     * @return \SprykerEco\Zed\Optivo\Business\Api\Response\ResponseConverterInterface
+     */
+    protected function createResponseConverter(): ResponseConverterInterface
+    {
+        return new ResponseConverter();
+    }
+
+    /**
+     * @return \SprykerEco\Zed\Optivo\Business\Api\Adapter\Http\HttpAdapterInterface
+     */
+    protected function createGuzzleAdapter(): HttpAdapterInterface
+    {
+        return new Guzzle($this->getConfig());
+    }
+
     /**
      * @return \SprykerEco\Zed\Optivo\Business\Api\Adapter\OptivoApiAdapterInterface
      */
-    protected function createOptivoApiAdapter()
+    protected function createOptivoApiAdapter(): OptivoApiAdapterInterface
     {
         return new OptivoApiAdapter(
-            $this->createGuzzle(),
+            $this->createGuzzleAdapter(),
             $this->createRequestUrlBuilder(),
             $this->createResponseConverter()
         );
     }
 
     /**
-     * @return \SprykerEco\Zed\Optivo\Business\Api\Adapter\Http\HttpAdapterInterface
+     * @return \SprykerEco\Zed\Optivo\Business\Mapper\Customer\CustomerMapperInterface
      */
-    protected function createGuzzle()
+    protected function createCustomerMapper(): CustomerMapperInterface
     {
-        return new Guzzle($this->getConfig());
+        return new CustomerMapper($this->getConfig(), $this->getLocaleFacade());
     }
 
     /**
-     * @return \SprykerEco\Zed\Optivo\Business\Api\Request\RequestUrlBuilderInterface
+     * @return \SprykerEco\Zed\Optivo\Business\Mapper\Customer\CustomerMapperInterface
      */
-    protected function createRequestUrlBuilder()
+    protected function createCustomerNewsletterMapper(): CustomerMapperInterface
     {
-        return new RequestUrlBuilder(
-            $this->getConfig()
-        );
+        return new CustomerNewsletterMapper($this->getConfig(), $this->getLocaleFacade());
     }
 
     /**
-     * @return \SprykerEco\Zed\Optivo\Business\Api\Response\ResponseConverterInterface
+     * @return \SprykerEco\Zed\Optivo\Business\Mapper\Order\OrderMapperInterface
      */
-    protected function createResponseConverter()
+    protected function createNewOrderMapper(): OrderMapperInterface
     {
-        return new ResponseConverter();
+        return new NewOrderMapper($this->getConfig(), $this->getMoneyFacade(), $this->getLocaleFacade());
+    }
+
+    /**
+     * @return \SprykerEco\Zed\Optivo\Business\Mapper\Order\OrderMapperInterface
+     */
+    protected function createPaymentNotReceivedMapper(): OrderMapperInterface
+    {
+        return new PaymentNotReceivedMapper($this->getConfig(), $this->getMoneyFacade(), $this->getLocaleFacade());
+    }
+
+    /**
+     * @return \SprykerEco\Zed\Optivo\Business\Mapper\Order\OrderMapperInterface
+     */
+    protected function createShippingConfirmationMapper(): OrderMapperInterface
+    {
+        return new ShippingConfirmationMapper($this->getConfig(), $this->getMoneyFacade(), $this->getLocaleFacade());
```

Comment:
I think it will look better if you indent this with new lines.

Reply:
Here string length is less then 120 chars, so it is ok